PWGHF/HFC/Tasks/taskCorrelationHfeHadrons.cxx

```diff
@@ -0,0 +1,304 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file taskCorrelationHfeHadrons.cxx
+/// \author Rashi Gupta <rashi.gupta@cern.ch>, IIT Indore
+/// \author Ravindra Singh <ravindra.singh@cern.ch>, IIT Indore
+
+#include "THnSparse.h"
+
+#include "DataFormatsEMCAL/AnalysisCluster.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/HistogramRegistry.h"
+#include "Framework/runDataProcessing.h"
+
+#include "Common/Core/PID/TPCPIDResponse.h"
+#include "Common/Core/RecoDecay.h"
+#include "Common/Core/TrackSelection.h"
+
+#include "Common/DataModel/Centrality.h"
+#include "Common/DataModel/EventSelection.h"
+#include "Common/DataModel/Multiplicity.h"
+#include "Common/DataModel/PIDResponse.h"
+#include "Common/DataModel/TrackSelectionTables.h"
+
+#include "PWGJE/DataModel/EMCALClusters.h"
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+using namespace o2::soa;
+
+struct HfTaskCorrelationHfeHadrons {
+  // Configurables
+
+  // Cluster information
+  Configurable<bool> clusterInfo{"clusterInfo", true, "EMCal cluster info before and after track match"};
+  // Run3 selection
+  Configurable<bool> isRun3{"isRun3", true, "Data is from Run3 or Run2"};
+  // Electron hadron correlation condition
+  Configurable<bool> pTCondition{"pTCondition", true, "Electron pT should be greater than Associate particle pT"};
```

Comment:
```suggestion
  Configurable<bool> ptCondition{"ptCondition", true, "Electron pT should be greater than Associate particle pT"};
```
- Fix capitalisation.
- The configurable is not the condition itself but a switch to use it or not, so it should be called something like `usePtCondition`.